src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+public class Rectangle extends Default {
+    private double side;
+    private double firstLeg;
+    private double secondLeg;
+
+    public Rectangle(String color, double side, double firstLeg, double secondLeg) {
+        super(color);
+        this.side = side;
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double getProperty() {
+        return firstLeg * secondLeg;
```

Comment:
The method 'getProperty()' seems to calculate the area of a rectangle using 'firstLeg' and 'secondLeg', which is inconsistent with the naming and purpose of the method. Consider renaming the method or adjusting its logic to align with its intended purpose.